### DISCUSSION TITLE
hanging out at validation step for several hours

### DISCUSSION THREAD

**Discussion by HuijeongJeong** [dandi/helpdesk#83] [Q&A] [answered]
Hi! I'm trying to upload nwb files and publish my dandiset (https://dandiarchive.org/dandiset/000351?pos=1). The upload was successful, but my dandiset says that it has asset validation errors from most of files (409 out of 426), and (probably because of this) the publish button is not available. It seems like the files are being validated for hours. What should I do? 
Thanks!
Huijeong

**Comment by mvandenburgh** [comment, marked as answer]
Thanks for the report! We're looking into this, it appears to be a bug on our side.

**Reply by HuijeongJeong** [reply]
I see, thank you!

**Reply by mvandenburgh** [reply]
@HuijeongJeong we've resolved the issue on our end. It looks like your assets have finished validating.

**Reply by namboodirilab** [reply]
Perfect! Thank you so much!!
